compiler/natives/src/runtime/runtime_test.go

```diff
@@ -0,0 +1,80 @@
+//go:build js
+// +build js
+
+package runtime
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/gopherjs/gopherjs/js"
+)
+
+func Test_parseCallstack(t *testing.T) {
+	tests := []struct {
+		name  string
+		input string
+		want  string
+	}{
+		{
+			name: "Chrome 96.0.4664.110 on Linux",
+			input: `at foo (eval at $b (https://gopherjs.github.io/playground/playground.js:102:11836), <anonymous>:25887:60)
```

Comment:
nit: this test case can probably be trimmed down in size by removing similar lines, which would help readability.

Reply:
Good point. I might do each line as a separate test, too.  I also think I can tidy up `parseCallstack` a bit more.